src/dioptra/restapi/db/alembic/versions/ad4f89b2288d_add_metrics_table.py

```diff
@@ -0,0 +1,31 @@
+"""add metrics table
+
+Revision ID: ad4f89b2288d
+Revises: 06e8176155bf
+Create Date: 2025-08-26 10:25:55.222239
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = 'ad4f89b2288d'
+down_revision = '06e8176155bf'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table('job_metrics',
+        sa.Column('job_resource_id', sa.BigInteger().with_variant(sa.Integer(), 'sqlite'), nullable=False),
+        sa.Column('name', sa.Text(), nullable=False),
+        sa.Column('value', sa.Float(), nullable=True),
+        sa.Column('step', sa.BigInteger().with_variant(sa.Integer(), 'sqlite'), nullable=False),
+        sa.Column('timestamp', sa.BigInteger().with_variant(sa.Integer(), 'sqlite'), nullable=False),
```

Comment:
This would need to be changed when changing this over to a datetime type rather than an integer type, see my comment in file `src/dioptra/restapi/db/models/jobs.py`